lib/hydra/works/models/concerns/collection_behavior.rb

```diff
@@ -0,0 +1,55 @@
+module Hydra::Works
+  module CollectionBehavior
```

Comment:
This module should `include Hydra::PCDM::CollectionBehavior` because it references `members` which is provided in that module.